templates/rustdoc/releases.html

```diff
@@ -0,0 +1,6 @@
+{% import "macros.html" as macros %}
+{% set target = "" %}
+{% set inner_path = releases[0].target_name ~ "/index.html" %}
+<ul>
```

Comment:
```suggestion
<ul class="pure-menu-list">
```

Without the class the list is offset inside the container

<img width="356" alt="image" src="https://user-images.githubusercontent.com/81079/235962513-0475d8e1-ba1d-43ea-8310-96e9957cae26.png">